README.md

```diff
@@ -71,6 +76,22 @@ Setting the cache directory and deserializer are optional but highly recommended
 * The deserializer allows for deserialization on a background thread. Without it, object deserialization will occur on the main (UI) thread. This can be bad for performance if the API response bodies are large.
 * The cache directory is to allow caching of requests. Without it, no requests will be cached and all other cache settings will be ignored.
 
+#### Configuration Builder for Apps with Only One Developer Account
+You can skip the need for [client credentials](#client-credentials-grant) or [code grant](#oauth-authorization-code-grant) requests if you provide an access token up front. With this access token you'll be able to make any requests that the access token's scope allows.
+
+You can generate an access token in the Authentication tab once you select your app from the [list here](https://developer.vimeo.com/apps).
+
+Once you have the access token, you can easily initialize your `VimeoClient` instance with the below builder.
+ 
+```java
+String accessToken = getString(R.string.access_token);
+return new Configuration.Builder(Vimeo.VIMEO_BASE_URL_STRING, accessToken);
+```
+
+After providing the access token, if you'd like to have access to the associated `User` object you'll need to make a call to `VimeoClient#fetchCurrentUser`. If you're using an account store, you can update the `VimeoAccount` with the new `User` object.
+
+*Note: You will not be able to log out of the account associated with the access token provided to the `Configuration.Builder`
```

Comment:
"This is because we wouldn't want anyone to accidentally invalidate/delete the token which is being used to authenticate users in a production application. You will still be able to delete the token via the web [developer console](https://developer.vimeo.com/apps/).
If this seems like restricting functionality, please log an issue to the [issue tracker](https://github.com/vimeo/vimeo-networking-java/issues)."